src/core/contrib.js

```diff
@@ -10,107 +10,86 @@ import { pub } from "core/pubsubhub";
 export const name = "core/contrib";
 
 function prop(prop) {
-  return function(o) {
-    return o[prop];
-  };
+  return o => o[prop];
 }
 
 function findUsers(...thingsToFind) {
-  var users = {};
-  thingsToFind.forEach(function(things) {
-    things.forEach(function(thing) {
+  const users = new Set();
+  for (const things of thingsToFind) {
+    for (const thing of things) {
       if (thing.user) {
-        users[thing.user.url] = true;
+        users.add(thing.user.url);
       }
-    });
-  });
-  return Object.keys(users);
+    }
+  }
+  return [...users];
 }
 
 function join(things) {
   if (!things.length) {
     return "";
   }
-  things = things.slice(0);
-  var last = things.pop();
-  var length = things.length;
-  if (length === 0) {
+  const length = things.length;
+  const last = things[length - 1];
+  if (length === 1) {
     return last;
   }
-  if (length === 1) {
-    return things[0] + " and " + last;
+  if (length === 2) {
+    return `${things[0]} and ${last}`;
   }
-  return things.join(", ") + ", and " + last;
+  return `${things.join(", ")}, and ${last}`;
 }
 
-function toHTML(urls, editors, element) {
-  return $.when
-    .apply(
-      $,
-      urls.map(function(url) {
-        return ghFetch(url);
-      })
-    )
-    .then(function(...args) {
-      var names = args
-        .map(function(user) {
-          user = user[0];
-          return user.name || user.login;
-        })
-        .filter(function(name) {
-          return editors.indexOf(name) < 0;
-        });
-      names.sort(function(a, b) {
-        return a.toLowerCase().localeCompare(b.toLowerCase());
-      });
-      $(element).html(join(names)).attr("id", null);
-    });
+async function toHTML(urls, editors, element) {
+  const args = await Promise.all(...urls.map(ghFetch));
```

Comment:
Might be a bug here: Promise.all is not variadic. You might want `Promise.all([...` instead.

Reply:
Apologies if I’m totally wrong about this. Maybe I’m misremembering?